src/core/component.ts

```diff
@@ -160,6 +160,11 @@ function initBaseComponent() {
                     ...this.$_getIntegrationOptions()
                 };
 
+                if ((options as any).dxKey) {
```

Comment:
We can define type `Record<string, any>` for the `options` and then we can get rid off `as any`.